package.json

```diff
@@ -14,7 +14,6 @@
     "node": "^16",
     "yarn": "^3"
   },
-  "type": "module",
```

Comment:
We can't use native ESM with 11ty very conveniently until: https://github.com/11ty/eleventy/issues/836

Before, it worked fine for us to use ESM for build tools, but 11ty was running entirely in a CJS context. Now we're using ts-node so that 11ty can import TS files, and it doesn't like operating in a mixed context with both ESM and CJS. See: https://github.com/TypeStrong/ts-node#err_require_esm